core/UIProfileGenerator.py

```diff
@@ -233,9 +233,20 @@ def generate_composite_attribute(self, profile_snapshot, attribute_defining_elem
                                                                       profile_snapshot.get("name"))
             return attribute_definition
         elif attribute_type == "CodeableConcept":
-            attribute_definition.referencedValueSet = self.parser.get_selectable_concepts(
-                attribute_defining_elements[-1],
-                profile_snapshot.get("name"))
+            if binding := predicate.get("binding"):
```

Comment:
Unused assigment. If you simply check if the return is None then the assigment is unnecessary